test/infrastructure/setup-management-cluster.sh

```diff
@@ -17,6 +17,8 @@ declare -a steps=(
 )
 
 for step in "${steps[@]}"; do
-  echo "$step"
-  source $step
+  echo "👉 $step"
+  source "${SCRIPTPATH}/$step"
 done
+
+echo "✅ Successfully setup test infrastructure"
```

Comment:
nit: newline

Reply:
I think the extra line is okay :) The sections are all broken up with an empty line and the script is small

Reply:
oh I mean you need a newline below :D